catalyst-gateway/bin/src/service/docs/stoplight_elements/mod.rs

```diff
@@ -17,11 +17,19 @@ const STOPLIGHT_UI_TEMPLATE: &str = r#"
     <style charset="UTF-8">{:style}</style>
     <script charset="UTF-8">{:script}</script>
   </head>
-  <body>
+  <body style="margin: 0; overflow: hidden;">
```

Comment:
The changes to this file should not be necessary because it's only used when cat-gateway displays its API document directly.

Reply:
removing this file from the commit